index.html

```diff
@@ -250,6 +250,31 @@ <h3 class="text-3">LET’S CONNECT</h3>
 
 
     </main>
+    <footer class="footer">
+        <div class="foot-image">
```

Comment:
### Your Background
![background-student](https://user-images.githubusercontent.com/93039359/187659467-5b372807-3150-4aba-806f-4cc4b55f8022.png)

### Figma Background
![background (copy)](https://user-images.githubusercontent.com/93039359/187659531-acfd8187-fb4e-415e-a5a3-82cdc600da21.png)

- [ ] Please make your background design look similar to this figma template so all the images shown in the red boxes will appear on your design as well :blush: